Node pool commands (#176)
banzaicloud/banzai-cli: cmd/docs/banzai_cluster_nodepool.md

## banzai cluster nodepool

> Work with cluster nodepools

Reviewer: ```suggestion
Manage  cluster node pools
```

### Synopsis

> Work with cluster nodepools

Reviewer: ```suggestion
Manage cluster node pools
```

### Options

```

>       --cluster int32         ID of cluster to nodepool

Reviewer: ```suggestion
      --cluster int32         ID of the cluster of which node pools to manage
```

>       --cluster-name string   Name of cluster to nodepool

Reviewer: ```suggestion
      --cluster-name string   Name of the cluster of which node pools to manage
```
